flutter/lib/l10n/app_en.arb

```diff
@@ -87,14 +87,17 @@
   "benchInfoObjectDetectionDesc": "Object detection draws bounding boxes around recognized objects in an input image, assigning each one a label. This is a common approach for identifying objects in photos, and automotive safety. The reference model is a Single Shot Detector based MobileDet model operating on the COCO 2017 validation dataset with a mean Average Precision (mAP) of 27.1 (95% of FP32 accuracy).\n\nMobileDet-SSD provides substantial improvements in the latency-accuracy trade-off by incorporating regular convolutions along with depthwise-separable convolutions in the search space for the object detection task, and effectively placing them in the network via neural architecture search (from: https://arxiv.org/pdf/2004.14525.pdf)",
   "benchInfoImageSegmentationDesc": "Semantic image segmentation divides an input image into labeled regions at pixel granularity (e.g., labeling all pixels associated with a car and labeling every pixel in the image), and is used for complex image scene parsing to power downstream image analysis and processing.\n\nThe reference model is the MOSAIC network, which was designed with current mobile processors and accelerators in mind, and uses a 512x512 input image resolution. The backbone feature extractor uses a modified version of MobileNet-v3.5-MultiHardware which was developed with Network Architecture Search (NAS) to optimize inference speed on various mobile processors and accelerators. Performance and accuracy are evaluated on the ADE20K validation dataset. The benchmark requires a mean Intersection Over Union (mIoU) value of 57.36% (96% of FP32 accuracy).\n\nMOSAIC uses an efficient multi-scale context encoder and a light-weight hybrid decoder to recover spatial details from aggregated information. MOSAIC produces state-of-the-art results balancing between model accuracy and speed across CPUs, GPUs, DSPs and NPUs.",
   "benchInfoLanguageProcessingDesc": "Question Answering finds the  best answer to an input question based on a body of text. The reference model, MobileBERT, is evaluated on the Stanford Question Answering Dataset (SQUAD) v1.1 Dev and achieves an F1-score of 90.0.\n\nMobileBERT is a thin, mobile-optimized version of the larger BERT (BERT_LARGE) network, equipped with bottleneck structures and a carefully designed balance between self-attention and feed-forward networks. Like BERT, it is task-agnostic, applicable for a variety of downstream NLP tasks, but for MLPerf we use the variant fine-tuned for Q&A.",
+  "benchInfoSuperResolutionDesc": "Super Resolution tbd.",
```

Comment:
@willc2010 @mkankana 
Can someone please provide me a description text for the Super Resolution task?

Reply:
"Super Resolution reconstructs a higher resolution(HR) image from lower resolution(LR) image. The reference model EDSR, is evaluated on OpenSR100 dataset."